core/controllers/base_test.py

```diff
@@ -303,38 +307,51 @@ def test_valid_pillow_path(self):
             Exception, 'Invalid path for oppia_tools library: invalid_path')
 
         def mock_os_path_join_for_pillow(*args):
+            """Mocks path for Pillow to test against an invalid path."""
             path = ''
             if args[1] == 'Pillow-6.0.0':
                 return 'invalid_path'
             else:
                 for arg in args:
-                    path += arg + '/'
+                    path = '/'.join(args)
```

Comment:
Why are you repeating this statement in a for loop?

/cc @oppia/dev-workflow-team @kevinlee12 Shouldn't pylint be catching "unused argument `arg`"?

Reply:
A mistake, reverted it!